src/client/eppo-client.ts

```diff
@@ -35,6 +36,7 @@ export interface IEppoClient {
 export default class EppoClient implements IEppoClient {
   constructor(
     private configurationRequestor: ExperimentConfigurationRequestor,
+    private poller: IPoller,
```

Comment:
Probably better to add the param at the end, so anyone implementing this constructor doesn't have to reshuffle arg order.